plugins/CoreHome/angularjs/history/history.service.js

```diff
@@ -67,7 +67,12 @@
                     searchObject[name] = searchObject[name][searchObject[name].length - 1];
                 }
 
-                searchString.push(name + '=' + encodeURIComponent(searchObject[name]));
+                var value = searchObject[name];
+                if (name != 'columns') { // the columns query parameter is not urldecoded in PHP code. TODO: this should be fixed in 3.0
```

Comment:
Yes it will be merged, after this merge we need to remove this `TODO`, that's hwy we need an issue for this or we will forget and not do it